libtest/src/main/java/tests/custom/transfer/AccountsWrong1.java

```diff
@@ -71,4 +72,17 @@ public String toString() {
                 "data=" + data +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Please, implement `VerifierState` (you can implement for the base `Accounts` class)

Reply:
Currently libtest main does not depend on lincheck submodule, so either this restriction can be removed or this classes can't implement ```VerifierState```

Reply:
Let's add this dependency, I don't see a problem.